excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/InvalidImportException.java

```diff
@@ -1,22 +1,17 @@
 package uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
  * Exception thrown if import of a Case Definition sheet fails due to missing data.
  *
  * @author Daniel Lam (A533913)
  */
 public class InvalidImportException extends RuntimeException {
-    private static final Logger logger = LoggerFactory.getLogger(InvalidImportException.class);
 
     public InvalidImportException() {
         // default constructor
     }
 
     public InvalidImportException(final String message) {
         super(message);
-        logger.error(message);
```

Comment:
with this change any validation error is no longer logged?

Reply:
reverted this change